modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/tree/io/BplusInnerIo.java

```diff
@@ -81,23 +84,24 @@ public final long getLeft(long pageAddr, int idx) {
     public final void setLeft(long pageAddr, int idx, long pageId) {
         assertPageType(pageAddr);
 
-        putLong(pageAddr, offset0(idx, SHIFT_LEFT), pageId);
+        writePartitionless(pageAddr + offset0(idx, SHIFT_LEFT), link(pageId, 0));
 
-        assert pageId == getLeft(pageAddr, idx);
+        assert pageId == getLeft(pageAddr, idx, partitionId(pageId));
     }
 
     /**
-     * Returns right page id for item.
+     * Returns right page ID for item.
```

Comment:
Please don't capitalize it. First of all, why would you touch a comment that has nothing to do with your changes. Second, "id" is grammatically correct.

Reply:
Fix it